pkg/loader/load.go

```diff
@@ -444,6 +450,79 @@ func (s *loaderImpl) execDDL(ddl *DDL) error {
 	return errors.Trace(err)
 }
 
+func (s *loaderImpl) processOracleDDL(ddl *DDL) error {
+	ddlStmt := ddl.SQL
+	newStmt := ""
+	if isTruncateTableStmt(ddlStmt) {
+		newStmt = fmt.Sprintf("BEGIN %s.do_truncate('%s.%s','');END;", ddl.Database, ddl.Database, ddl.Table)
+	} else {
+		ok, astStmt := isTruncateTablePartitionStmt(ddlStmt)
+		if ok {
+			partitions := astStmt.Specs[0].PartitionNames
+			newStmt = fmt.Sprintf("BEGIN %s.do_truncate('%s.%s','%s');END;", ddl.Database, ddl.Database, ddl.Table, partitions[0].O)
```

Comment:
Is that okay if we only alter one partition here?

Reply:
delete it